balboa-core/src/main/scala/com/socrata/balboa/metrics/data/impl/Cassandra11Util.scala

```diff
@@ -103,16 +104,25 @@ object Cassandra11Util extends StrictLogging {
     logger.info("Setting Cassandra socket timeout to '{}'", sotimeout.toString)
     logger.info("Using keyspace '{}'", keyspace)
 
+    val connectionPoolConfiguration = new ConnectionPoolConfigurationImpl("BalboaPool")
+      .setConnectTimeout(sotimeout)
+      .setTimeoutWindow(sotimeout)
+      .setMaxConnsPerHost(connections)
+      .setSeeds(seeds)
+
+    // Set local DC as side-effect if specified in configuration.
+    // If the local datacenter is specified it will limit the driver
+    // to only make connections to the Cassandra nodes in the datacenter
+    // and prevent this service from unintentionally reaching across a
+    // VPN to connect to a Cassandra node.
+    datacenter.foreach(connectionPoolConfiguration.setLocalDatacenter)
```

Comment:
Do you have some reason to believe that it is possible to set multiple local datacenters? When I look at the implementation of `setLocalDatacenter`, it doesn't seem to accumulate them, it seems to just set one object level field. And `getLocalDatacenter` just returns a String.